jp/src/main/resources/providers/hospitals_jp.csv

```diff
@@ -0,0 +1,9 @@
+,id,name,address,city,state,zip,county,phone,type,ownership,emergency,quality,LAT,LON
+0,10001,Tokyo Medical Hospital,"2-5-1 Higashigaoka, Meguro-ku",Minato,Tokyo,100-8602,Tokyo,03-3411-0111,National Hospital Organization,National Hospital Organization,Yes,5,35.6895,139.69171
+,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,
+,,,,,,,,,,,,,139.69171,
```

Comment:
You should delete these extra rows with null values, they'll probably cause errors during runtime.